ui/imports/shared/views/profile/ProfileShowcaseView.qml

```diff
@@ -175,7 +169,7 @@ Control {
                     components: [
                         StatusIcon {
                             anchors.verticalCenter: parent.verticalCenter
-                            visible: model.walletType === Constants.watchWalletType
+                            //visible: model.walletType === Constants.watchWalletType
```

Comment:
Is this a leftover?

Reply:
leftover, we already check this when filling the showcase model